example/example/tests/test_grapple.py

```diff
@@ -4,10 +4,7 @@
 
 import wagtail_factories
 
-if sys.version_info >= (3, 7):
-    from builtins import dict as dict_type
-else:
-    from collections import OrderedDict as dict_type
+from builtins import dict as dict_type
```

Comment:
I had to remove this check because of failing tests.
I am not sure whether we should keep it, what do you think ?